src/api/index.js

```diff
@@ -1,882 +1,22 @@
-const mockedData = {
-  kind: 'youtube#searchListResponse',
-  etag: 'LRviZfd_p3HDDD2uBk5Qv7zaEQU',
-  nextPageToken: 'CBkQAA',
-  regionCode: 'MX',
-  pageInfo: {
-    totalResults: 2323,
-    resultsPerPage: 25,
+const apiUrlBuilder = (endpoint, queryParams) => {
+  return `https://www.googleapis.com/youtube/v3/${endpoint}?${new URLSearchParams({
+    key: process.env.REACT_APP_GOOGLE_API_KEY,
+    part: 'snippet',
+    type: 'video',
+    maxResults: 25,
+    ...queryParams,
+  })}`;
+};
+
+export default {
+  searchVideos: (searchValue) => {
+    return fetch(apiUrlBuilder('search', { q: searchValue })).then((response) =>
+      response.json()
+    );
+  },
+  getRelatedVideos: (videoId) => {
+    return fetch(
+      apiUrlBuilder('search', { relatedToVideoId: videoId })
+    ).then((response) => response.json());
```

Comment:
You could remove the `return`, but, if you believe it is more clear with the return there it is good 😊